client/dashboard/src/services/article-api-service.js

```diff
@@ -0,0 +1,13 @@
+const artcileEndpoint = 'http://localhost:4000/articles';
+
+exports.getArticles = (state, stateSetter) => {
+  fetch(artcileEndpoint, {
```

Comment:
```suggestion
  fetch(artcileEndpoint)
    .then(res => res.json())
    .then(articles => ........
```
For GET method and with just Contet-Type: application/json you don't need to pass options parameter.